frontend/public/components/secrets/create-secret.tsx

```diff
@@ -5,9 +5,11 @@ import { Helmet } from 'react-helmet';
 import { Link } from 'react-router-dom';
 
 import { k8sCreate, k8sUpdate, K8sResourceKind } from '../../module/k8s';
-import { ButtonBar, Firehose, history, kindObj, StatusBox, FileInput } from '../utils';
+import { ButtonBar, Firehose, history, kindObj, StatusBox } from '../utils';
 import { formatNamespacedRouteForResource } from '../../ui/ui-actions';
 import { WebHookSecretKey } from '../secret';
+import { AsyncComponent } from '../utils/async';
+import withDragDropContext from '../utils/withDragDropContext';
```

Comment:
This is going to pull in react-dnd to the main bundle. I think this needs to be in file-input.tsx if we can move it.

Reply:
if we move it to file-input.tsx then we wont be able to have more then one instances of the `DropableFileInput` component, since everytime the HTML5Backend will be loaded.

Reply:
but you are right, we need to load it async

Reply:
It's a singleton, though. If that doesn't work, then something is wrong in how we defined `withDragDropContext`